x/oracle/keeper/reward.go

```diff
@@ -30,17 +49,6 @@ func (k Keeper) RewardBallotWinners(
 	voteTargets []string,
 	ballotWinners []types.Claim,
 ) {
-	// sum weight of the claims
-	var ballotPowerSum int64
-	for _, winner := range ballotWinners {
-		ballotPowerSum += winner.Weight
-	}
-
-	// return if the ballot is empty
-	if ballotPowerSum == 0 {
-		return
-	}
```

Comment:
Is there a case we're not handling here?

Reply:
Yeah I was checking for the zero ballots case in the `smallestMissCountInBallot` method but I think I should actually do that check in `RewardBallotWinners` and return right away if its empty. I made this update in [ff38d22](https://github.com/ojo-network/ojo/pull/53/commits/ff38d222f64cb81dc8f0b1c6416977c47c8e28b1)

Reply:
perfect